src/components/app/app.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+
+import Form from '../form/form';
+import ItemList from '../item-list/item-list'
+
+import './app.css';
+
+export default class App extends Component {
+
+  state = {
+    data: []
+  }
+
+  // simple counter to make different dummy requests 
+  count = 1;
```

Comment:
do we need `let` here?

Reply:
When we initialize a variable inside ES6 classes, we don't need to use let is this case. It will throw an error.